### DISCUSSION TITLE
[BUG] search bar not working fabric 1.19 (was told to make a discussion instead of an issue)

### DISCUSSION THREAD

**Discussion by alexioron** [shedaniel/RoughlyEnoughItems#1010] [Q & A] [answered]
what the title says, i was told to make a discussion instead of an issue by the owner so i'm doing that now

logs: https://gist.github.com/alexioron/f26d58eb97f176c3af64ff1848977795

**Comment by shedaniel** [comment, marked as answer]
Your issue is caused by Fabric Seasons, you could put the blame onto REI for asynchronously searching, but I feel like Fabric Seasons should add a check on their end instead.

**Reply by alexioron** [reply]
thanks a bunch, good thing it wasnt caused by an actual content mod or i'd be in some trouble
